src/main/java/org/eclipse/microprofile/starter/rest/APIService.java

```diff
@@ -265,6 +275,30 @@ public Response supportMatrixServersV3(String ifNoneMatch) {
         return Response.ok(serversAndSpecsDescriptions, MediaType.APPLICATION_JSON_TYPE).tag(serversToOptionsEtag).build();
     }
 
+    public Response supportMatrixServersv6(String ifNoneMatch) {
+        if (ifNoneMatch != null) {
+            if (serversToOptionsEtag.toString().equals(ifNoneMatch)) {
+                return Response.notModified().build();
+            }
+        }
+        List<SupportedServer> servers = new ArrayList<>(serversToOptions.keySet());
+        Collections.shuffle(servers);
+        Map<SupportedServer, List<ServerOptions>> rndServersToOptions = new LinkedHashMap<>(servers.size());
+        for (SupportedServer s : servers) {
+            List<ServerOptions> options = serversToOptions.get(s);
+            // Without the buildTool
+            List<ServerOptions> newOptions = options.stream()
```

Comment:
Why not include buildTool?

Reply:
This is the 'old' version of the response. The buildTool element was not known in those cases. (and clients might not  handle it properly, the additional property within JSON)